### PR TITLE
2023-18-07 changelog, images and video

### PR DESCRIPTION
**Pull request by alexfrancoeur**

### CODE REVIEW

**Review by philkra — APPROVED** [review]
LGTM

File: 2023-07-18.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Our docs just got a major face lift, matching the rest of our branding. Other improvements include:
+    - The documentation content is now [public on GitHub](https://github.com/xataio/mdx-docs) and open for community edits.
+    - Website search was rebuilt and will now output results from the blog alongside documentation.
+    - Grammar and syntax highlighting in code snippets are improved.
+
+<ArticleImage
+  src="https://github.com/xataio/mdx-changelog/raw/main/images/2023-07-18-01-docs.png"
+  alt="A new style for the Xata documentation"
+  caption="A new style for the Xata documentation"
+/>
```
Comment (snide) [inline 2023-07-18.mdx:14]:
```suggestion
![A new style for the Xata documentation](images/2023-07-18-01-docs.png)
```

You can use markdown format for this now, and reference the image directly

File: 2023-07-18.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    src="https://github.com/xataio/mdx-changelog/raw/main/images/2023-07-18-02-delete.mov" 
+/>
+
+- Display an object deprecation notice where appropriate throughout the Xata app
+
+<ArticleImage
+  src="https://github.com/xataio/mdx-changelog/raw/main/images/2023-07-18-03-object.png"
+  alt="Object type deprecation warnings"
+  caption="Object type deprecation warnings"
+/>
```
Comment (snide) [inline 2023-07-18.mdx:34]:
```suggestion
![bject type deprecation warnings](images/2023-07-18-03-object.png)
```

**Review by snide — APPROVED** [review]
Formatting. Feel free to merge with commits.

**Comment by alexfrancoeur** [comment]
Thanks @snide and @philkra !

**Pull request merged**